packages/core/src/Keymap.ts

```diff
@@ -12,7 +12,8 @@ interface ShortcutPattern {
 
 interface ShortcutItem {
     pattern: ShortcutPattern;
-    boundCommand: BoundCommand;
+    // if there is no bound command, it means that we want to remove the ShortcutItem.
```

Comment:
I think this might be one of these few cases where explicitely typing `undefined` might be useful. I would make it explicit in the `Shortcut` interface of `JWEditor` (and thus not having it extend `BoundCommand`), like this:
```typescript
export interface Shortcut {
    platform?: Platform;
    pattern: string;
    commandId: CommandIdentifier | undefined;
    commandArgs?: CommandArgs;
}
```
I think it might help as a "documentation" for creating shortcuts in plugins. (Just to be clear, it is fine that it is optional in the interface right here.)